packages/react/src/components/modal/dialog/modal-dialog.tsx

```diff
@@ -0,0 +1,144 @@
+import { Fragment, ReactElement, Ref, useMemo, useRef, VoidFunctionComponent } from 'react';
+import { useTranslation } from '../../../i18n/use-translation';
+import { v4 as uuid } from '../../../utils/uuid';
+import { useDeviceContext } from '../../device-context-provider/device-context-provider';
+import { Heading } from '../../heading/heading';
+import { Modal } from '../modal';
+import {
+    ButtonContainer,
+    CancelButton,
+    ConfirmButton,
+    StyledHeadingWrapperComponent,
+    Subtitle,
+    TitleIcon,
+} from './styled';
+import { DialogType, ModalDialogProps } from './types';
+
+const ModalRoles: Record<DialogType, string> = {
```

Comment:
Ca ne devrait pas etre PascalCase puisque ce n'est pas un enum ou equivalent (constants pour ces valeurs), c'est plutot un mapping entre 2 types de valeurs.

```suggestion
const modalRoles: Record<DialogType, string> = {
```